exercise-src/react-native/13-http-requests/01-problem/src/services/pmo/restaurant/hooks.ts

```diff
@@ -5,9 +5,9 @@ import { State } from "./interfaces"
 const baseUrl = process.env.PMO_API
 
 interface StatesResponse {
+  isPending: boolean
   data: State[] | undefined
   error: Error | undefined
-  isPending: boolean
```

Comment:
Why this change? They’re in alphabetical order right now and that’s nice for scanning property names.

Reply:
Except for very long lists, I don't find alphabetical a very useful way to organize things. I usually order things by when are used, how often they're needed, the value they bring, etc. In this case: `isPending` is metadata about the rest of the items. (If there was a `status`, I'd put it first since isPending is derived from it.) `It will also be needed more frequently than the other two.